### PR TITLE
Update UG to use user-centric style

### PR DESCRIPTION
**Pull request by Daphne789**
Updated UG features: riskprofile, add, clone, calendar, clear

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -280,7 +280,7 @@ If a required parameter is missing (e.g., NAME, EMAIL)
 
   ![add format](images/add-UG/addmissingparam.png)
 
-If a parameter is provided in an invalid format (e.g., an invalid email address), an error message should indicate the invalid format.
+If a parameter you provided in an invalid format (e.g., an invalid email address), an error message should indicate the invalid format.
```
Comment (bhnuka) [inline docs/UserGuide.md]: add an 'is' after provided (i.e. "If a parameter you provided **_is_** in an invalid format..."

**Review by bhnuka — COMMENTED** [review]
One small nitpick! Other than that LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (Daphne789) [reply docs/UserGuide.md]: Thank you, Bhanuka!

Comment (bhnuka) [reply docs/UserGuide.md]: LGTM!

**Pull request merged**